src/cli/msig/index.ts

```diff
@@ -164,6 +170,134 @@ export default async function msig(args: string[]) {
+
+            if (!contractName) {
+                throw new Error('Contract name must be provided for deploy-contract proposal');
+            }
+
+            const contractInfo = {
+                account: contractName,
+                contractDir: path.join(__dirname, `../../Tonomy-Contracts/contracts/${contractName}`),
+            };
+
+            const { wasmPath, abiPath } = getDeployableFilesFromDir(contractInfo.contractDir);
+
+            const wasmFile = fs.readFileSync(wasmPath);
+            const abiFile = fs.readFileSync(abiPath, 'utf8');
+            const wasm = wasmFile.toString(`hex`);
+
+            // 2. Prepare SETABI
+            const abi = JSON.parse(abiFile);
+            const abiDef = ABI.from(abi);
+            const abiSerializedHex = Serializer.encode({ object: abiDef }).hexString;
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            const actions: ActionData[] = [];
+
+            // Prepare SETCODE action
+            const setCodeAction: ActionData = {
+                account: 'tonomy',
+                name: 'setcode',
+                authorization: [
+                    {
+                        actor: contractName.toString(),
+                        permission: 'active',
+                    },
+                ],
+                data: {
+                    account: contractName.toString(),
+                    vmtype: 0,
+                    vmversion: 0,
+                    code: wasm,
+                },
+            };
+
+            actions.push(setCodeAction);
+
+            // Prepare SETABI action
+            const setAbiAction: ActionData = {
+                account: 'tonomy',
+                name: 'setabi',
+                authorization: [
+                    {
+                        actor: contractName.toString(),
+                        permission: 'active',
+                    },
+                ],
+                data: {
+                    account: contractName.toString(),
+                    abi: abiSerializedHex,
+                },
+            };
+
+            actions.push(setAbiAction);
+
+            const proposalHash = await createProposal(
+                proposer,
+                proposalName,
+                actions,
+                privateKey,
+                newGovernanceAccounts
+            );
+
+            if (test) await executeProposal(proposer, proposalName, proposalHash);
+        } else if (proposalType === 'eosio.code-permission') {
+            for (const account of addCodePermissionTo) {
+                const accountName = TonomyUsername.fromUsername(
+                    account,
+                    AccountType.PERSON,
+                    getSettings().accountSuffix
+                );
+
+                const accountInfo = await getAccountInfo(accountName);
+                const ownerPermission = accountInfo.permissions.find((perm) => perm.perm_name.toString() === 'owner');
```

Comment:
accountInfo.getPermission("owner") will do it this will also throw error if doesn't exist.